app/src/main/java/com/pvp/app/ui/screen/settings/Settings.kt

```diff
@@ -78,6 +85,35 @@ fun CategoryRow(
     )
 }
 
+@Composable
+fun ReminderSetting() {
+    Row {
+        Text(
+            text = "Reminder time",
+            style = MaterialTheme.typography.titleLarge,
+        )
+    }
+
+    Text(
+        modifier = Modifier.padding(vertical = 8.dp),
+        text = "Selected (minutes): $reminderTimeMinutes",
+    )
+
+    Slider(
+        value = reminderTimeMinutes.toFloat(),
+        onValueChange = { reminderTimeMinutes = it.toInt() },
+        valueRange = 1f..60f,
+        steps = 59,
+        modifier = Modifier.padding(vertical = 8.dp)
+    )
+
+    Divider(
```

Comment:
I believe this is deprecated. If I am right, IDE should give warnings about it and you should listen to your IDE and refactor code into suggested better alternatives.